templates/managed/apps/index.html

```diff
@@ -16,7 +16,7 @@
       <h1>Performant open source with Managed Apps</h1>
       <p class="p-heading--4">One vendor, easy deployment, guaranteed uptime</p>
       <p>
-        <a href="/managed/contact-us" class="p-button--positive js-invoke-modal" data-testid="interactive-form-link">Free deployment assessment</a>
+        <a href="#get-in-touch" class="p-button--positive js-invoke-modal" data-testid="interactive-form-link" data-form-location="/shared/forms/interactive/managed-apps.html" data-form-id="3545" data-lp-id="3828" data-return-url="https://ubuntu.com/managed/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">Free deployment assessment</a>
```

Comment:
Currently we can only have one modal form per page, as a result this form is sent with the incorrect form id (3545). If we want to have two different forms, one will have to be a static form on a different page.

Reply:
@petesfrench based on what I see in the code and on other pages. It's possible to have multiple modal forms per page. As seen in the dynamic-forms script: https://github.com/canonical/ubuntu.com/blob/main/static/js/src/dynamic-forms.js#L19-#L23

It first checks if the clicked button has form details. If it doesn't then it loads the default form, which is the one at the bottom of the file.

Reply:
So the reason I push back is because both forms are being sent to the same form ID, and this is causing [an error on sentry](https://sentry.is.canonical.com/canonical/ubuntu-com/issues/20827/?query=3545) meaning the form `managed-apps-contact-us` to not be sent to marketo correctly. Possibly the [campaign for 3545](https://engage-sj.marketo.com/?munchkinId=066-EOV-335#/mktForm/3545/overview/details) isn't setup to receive the input `company`

Reply:
Hmmmm, that's weird, I tested it on here, on the demo, and I saw my entries on marketo. So what do you suggest we do?

Reply:
Just tested it again and it appears to working :raised_hands: looks like 'company' was added to the form definition